xmtp_proto/buf.gen.yaml

```diff
@@ -13,11 +13,16 @@ plugins:
   - plugin: buf.build/community/neoeinstein-tonic
     out: src/gen
     opt:
-      - no_server=true
+      - no_server=false
```

Comment:
We now need to generate servers everywhere. We may want to add a second feature to the server attributes so that we can toggle them off as needed.